Resources/config/services.yaml

```diff
@@ -8,6 +8,15 @@ services:
             - controller.service_arguments
             - container.service_subscriber
 
+    Orbitale\Bundle\CmsBundle\Controller\PostsController:
+        public: true
+        autowire: true
```

Comment:
I do not recommend auto-wiring in open-source bundles/packages, because anyone could change the definition of any service, so to make sure everything is kept as-is and arguments are "harder" to override, I recommend disabling autowiring whenever possible.

Also, you already defined the arguments, so auto-wiring is not necessary.

Reply:
Got it

Reply:
Not possible, cause tests get failed